common/djangoapps/student/tasks.py

```diff
@@ -6,12 +6,46 @@
 from boto.exception import NoAuthHandlerFound
 from celery.exceptions import MaxRetriesExceededError
 from celery.task import task  # pylint: disable=no-name-in-module, import-error
+from django.apps import apps
 from django.conf import settings
 from django.core import mail
+from django.contrib.auth.models import User
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 log = logging.getLogger('edx.celery.task')
 
 
+@task()
+def send_payment_link_to_parent(user_id):
+    """
+    Sending an payment link email to the parent.
+    """
+    ParentProfile = apps.get_model('tedix_ro', 'ParentProfile')
+    try:
+        user = User.objects.get(pk=user_id)
+        parent = user.parentprofile
+    except ParentProfile.DoesNotExist:
+        log.error("send_payment_link_to_parent: This user has no parentprofile.")
+    except User.DoesNotExist:
+        log.error("send_payment_link_to_parent: User with this ID does not exist.")
+    
+    from_address = configuration_helpers.get_value(
+        'email_from_address',
+        settings.DEFAULT_FROM_EMAIL
+    )
+    if user.is_active and parent:
+        for student in parent.students.filter(paid=False, user__is_active=True):
+            mail.send_mail(
+                    'pay link',
+                    'message',
+                    from_address,
+                    [user.email]
+                )
+            log.info("send_payment_link_to_parent: Email has been sent to User {user_email}".format(
+                user_email=dest_addr
```

Comment:
where do you define `dest_addr` variable? and why don't you use just `user.email`?